packages/components/icon-button/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@spark-ui/icon-button",
+  "version": "1.0.0",
+  "description": "Icon button renders an icon within in a button",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "vite build"
+  },
+  "dependencies": {
+    "@spark-ui/button": "^1.6.0",
+    "class-variance-authority": "0.4.0"
```

Comment:
```suggestion
    "@spark-ui/button": "1",
    "class-variance-authority": "0.5.2"
```

Reply:
I think that for our component @acd02 said that is better to use caret version

Reply:
BTW I think that would be nice to create a wrapper around cva 🤔 WDYT ?

Reply:
why? if its necessary to upgrade the caret version it means its not exactly retrocompatible

Reply:
I thought we all agreed that we should use the caret version, whether we choose something like `^1.4.0` or simply `1`, no?